PurchasesTests/Purchasing/StoreKitRequestFetcherTests.swift

```diff
@@ -141,7 +102,7 @@ class StoreKitRequestFetcher: XCTestCase {
     
     func testOnlyCreatesOneRefreshRequest() {
         setupFetcher(fails: false)
-        expect(self.factory?.requests).to(haveCount(2))
+        expect(self.factory?.requests).to(haveCount(1))
```

Comment:
Weird that this changed, does the migration fix a bug related to this?

Reply:
not really, but I should have added a comment here to explain: 
This class used to be responsible for both fetching receipts and products, so removing the products part of it makes the class only perform one request instead of 2

Reply:
Ah, makes sense 👍

Reply:
this is the line that causes this change https://github.com/RevenueCat/purchases-ios/pull/602/files#diff-6e795e719a9b41fcd5735980fd2532f93d255a731e79dffe8e3af58edadbeb10L67